packages/http/src/validator/validators/security/handlers/none.ts

```diff
@@ -0,0 +1,12 @@
+import { get } from 'lodash';
+
+import { when } from './utils';
+import { Dictionary } from '@stoplight/types';
+import { IHttpRequest } from '../../../../types';
+
+// Makes sure there aren't any auth headers
+function isNoAuth(inputHeaders: Dictionary<string>) {
+  return get(inputHeaders, 'authorization') == undefined;
+}
+
+export const none = (input: Pick<IHttpRequest, 'headers' | 'url'>) => when(isNoAuth(input.headers || {}), 'None');
```

Comment:
Do y'all think I should try to make this more elaborate? Like, somehow invert the other security schemes to check there's no API Key?

Reply:
Ahhh... to check and see if they're trying to pass something that matches an api key as a query param? I'm guessing that's why you're taking the url in with the headers?

I don't think so.  Extraneous query params just tend to be tolerated and ignored, I guess, and while we could probably identify the case that it's specifically "trying" to use an API Key when it shouldn't I imagine that would only happen (in our tooling) if you had a improper security scheme AND combination that included Optional... that's the only time we try to mix / match in terms of headers / query params for different schemes.